tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -177,24 +174,12 @@ def number_of_columns(self):
 
     class InstalledApp(PageRegion):
```

Comment:
We don't need switch_to_root_element() in this class, which makes it way simpler. Cool! :)